File: src/main/java/com/nayasis/simplelauncher/jpa/repository/ConfigRepository.java
package com.nayasis.simplelauncher.jpa.repository;

import com.nayasis.simplelauncher.jpa.entity.ConfigEntity;
import com.nayasis.simplelauncher.jpa.entity.LinkEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ConfigRepository extends JpaRepository<ConfigEntity,Long> {

    ConfigEntity findByKey( String key );

    void deleteByKey( Long id );

}

File: src/main/kotlin/com/github/nayasis/simplelauncher/jpa/repository/ConfigRepository.kt
package com.github.nayasis.simplelauncher.jpa.repository

import com.github.nayasis.simplelauncher.jpa.entity.Config
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ConfigRepository: JpaRepository<Config,String> {
}